include/vapor/VolumeRenderer.h

```diff
@@ -58,10 +58,10 @@ class RENDER_API VolumeRenderer : public Renderer
     virtual std::string _getDefaultAlgorithmForGrid(const Grid *grid) const;
     bool _needToSetDefaultAlgorithm() const;
     
-    unsigned int _VAO = NULL;
-    unsigned int _VBO = NULL;
-    unsigned int _VAOChunked = NULL;
-    unsigned int _VBOChunked = NULL;
+    unsigned int _VAO = (int)NULL;
+    unsigned int _VBO = (int)NULL;
+    unsigned int _VAOChunked = (int)NULL;
+    unsigned int _VBOChunked = (int)NULL;
     VolumeAlgorithm *_algorithm = NULL;
```

Comment:
We decided to adtop c++11 convention, so NULL is discouraged, and nullptr is encouraged.

Reply:
Fixed in #b2e2b92